docs/build/configure/parts-and-remotes.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Machine Architecture: Parts, Sub-Parts and Remotes"
+title: "Machine Architecture: Parts, Sub-Parts, and Remotes"
```

Comment:
Maybe just this, since the title is now too long? I suppose the sub part could be considered as part of part? 
```suggestion
title: "Machine Architecture: Parts and Remotes"
```

Reply:
I like having sub-parts in there for clarity and emphasis. I'd rather leave it in that adhere to oxford commas. Could just close this PR.